src/sentry/buffer/redis.py

```diff
@@ -86,8 +86,10 @@ class RedisOperation(Enum):
     SORTED_SET_GET_RANGE = "zrangebyscore"
     SORTED_SET_DELETE_RANGE = "zremrangebyscore"
     HASH_ADD = "hset"
+    HASH_ADD_BULK = "hmset"
```

Comment:
According to https://redis.io/docs/latest/commands/hmset/, this command is deprecated after 4.0 and can be replaced with `hset` which can take multiple key value pairs.
I'm not what version of redis we use tho

Reply:
yeah, i had initially tried the `hset` with `mapping` syntax and it threw errors, so we'll need to use hmset until then.

Reply:
ah ok, could we add a comment somewhere then referencing the future deprecation?